flow-typed/debugger-html.js

```diff
@@ -0,0 +1,216 @@
+  // @flow
```

Comment:
I added this file to share the debugger types. Going forward, i think it would be nice to have the [lib-def](https://flow.org/en/docs/libdefs/) exported by the debugger, but that is not a blocker for this otherwise massive PR.